src/ontology/mondo-edit.obo

```diff
@@ -212080,38 +212080,32 @@ subset: ordo_morphological_anomaly {source="Orphanet:2300"}
 subset: orphanet_rare {source="Orphanet:2300"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
-synonym: "CID-MIA/early-onset IBD" RELATED []
-synonym: "combined immunodeficiency-enteropathy spectrum" RELATED []
-synonym: "familial intestinal polyatresia syndrome" RELATED []
-synonym: "gastrointestinal defects and immunodeficiency syndrome" RELATED [MONDO:Lexical]
-synonym: "GIDID" RELATED ABBREVIATION [MONDO:Lexical]
 synonym: "intestinal atresia multiple" RELATED [GARD:0003013]
 synonym: "intestinal atresia, multiple" RELATED []
-synonym: "intestinal atresia, multiple and/or inflammatory bowel disease with or without immunodeficiency" RELATED []
-synonym: "multiple gastrointestinal atresias" EXACT [DOID:14671]
-synonym: "multiple intestinal atresia" EXACT CLINGEN_LABEL [DOID:14671, Orphanet:2300]
-xref: DOID:14671 {source="MONDO:equivalentTo"}
+synonym: "isolated multiple intestinal atresia" EXACT [Orphanet:2300]
+synonym: "multiple gastrointestinal atresias" RELATED [DOID:14671]
+synonym: "multiple intestinal atresia" EXACT CLINGEN_LABEL [Orphanet:2300]
+xref: DOID:14671 {source="MONDO:relatedTo"}
 xref: GARD:3013 {source="MONDO:GARD"}
 xref: ICD10CM:Q43.8 {source="Orphanet:2300", source="Orphanet:2300/attributed", source="Orphanet:2300/ntbt"}
 xref: ICD10CM:Q82.8 {source="Orphanet:436252/attributed", source="Orphanet:436252/ntbt", source="Orphanet:436252"}
 xref: ICD9:751.8 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MedDRA:10028210 {source="Orphanet:2300", source="Orphanet:2300/e"}
-xref: MEDGEN:65090 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:C562441 {source="MONDO:equivalentTo", source="DOID:14671"}
+xref: MEDGEN:65090 {source="MONDO:relatedTo", source="MONDO:MEDGEN"}
```

Comment:
Note that this change will be removed during the automated EMC update. Please tag Megan in the issue so she can update her files (this is how we get this change to be permanent)